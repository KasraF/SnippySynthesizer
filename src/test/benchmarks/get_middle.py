def get_middle(s):
    rs = s[(len(s)-1)//2:len(s)//2+1]
    return rs

get_middle("test") == "es"
get_middle("testing") == "t"
get_middle("middle") =="dd"
get_middle("A") == "A"
get_middle("of") == "of"