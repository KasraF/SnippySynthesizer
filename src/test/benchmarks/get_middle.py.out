[
  {
    "1": [
      "rs"
    ]
  },
  {
    "1": [
      {
        "time": 0,
        "#": "",
        "$": "",
        "s": "'test'",
        "lineno": 1,
        "next_lineno": 2
      },
      {
        "time": 3,
        "#": "",
        "$": "",
        "s": "'testing'",
        "lineno": 1,
        "prev_lineno": "R2",
        "next_lineno": 2
      },
      {
        "time": 6,
        "#": "",
        "$": "",
        "s": "'middle'",
        "lineno": 1,
        "prev_lineno": "R2",
        "next_lineno": 2
      },
      {
        "time": 9,
        "#": "",
        "$": "",
        "s": "'A'",
        "lineno": 1,
        "prev_lineno": "R2",
        "next_lineno": 2
      },
      {
        "time": 12,
        "#": "",
        "$": "",
        "s": "'of'",
        "lineno": 1,
        "prev_lineno": "R2",
        "next_lineno": 2
      }
    ],
    "2": [
      {
        "time": 1,
        "#": "",
        "$": "",
        "s": "'test'",
        "rs": "'es'",
        "lineno": 2,
        "prev_lineno": 1,
        "next_lineno": "R2"
      },
      {
        "time": 4,
        "#": "",
        "$": "",
        "s": "'testing'",
        "rs": "'t'",
        "lineno": 2,
        "prev_lineno": 1,
        "next_lineno": "R2"
      },
      {
        "time": 7,
        "#": "",
        "$": "",
        "s": "'middle'",
        "rs": "'dd'",
        "lineno": 2,
        "prev_lineno": 1,
        "next_lineno": "R2"
      },
      {
        "time": 10,
        "#": "",
        "$": "",
        "s": "'A'",
        "rs": "'A'",
        "lineno": 2,
        "prev_lineno": 1,
        "next_lineno": "R2"
      },
      {
        "time": 13,
        "#": "",
        "$": "",
        "s": "'of'",
        "rs": "'of'",
        "lineno": 2,
        "prev_lineno": 1,
        "next_lineno": "R2"
      }
    ],
    "R2": [
      {
        "time": 2,
        "#": "",
        "$": "",
        "s": "'test'",
        "rs": "'es'",
        "lineno": "R2",
        "prev_lineno": 2,
        "rv": "'es'",
        "next_lineno": 1
      },
      {
        "time": 5,
        "#": "",
        "$": "",
        "s": "'testing'",
        "rs": "'t'",
        "lineno": "R2",
        "prev_lineno": 2,
        "rv": "'t'",
        "next_lineno": 1
      },
      {
        "time": 8,
        "#": "",
        "$": "",
        "s": "'middle'",
        "rs": "'dd'",
        "lineno": "R2",
        "prev_lineno": 2,
        "rv": "'dd'",
        "next_lineno": 1
      },
      {
        "time": 11,
        "#": "",
        "$": "",
        "s": "'A'",
        "rs": "'A'",
        "lineno": "R2",
        "prev_lineno": 2,
        "rv": "'A'",
        "next_lineno": 1
      },
      {
        "time": 14,
        "#": "",
        "$": "",
        "s": "'of'",
        "rs": "'of'",
        "lineno": "R2",
        "prev_lineno": 2,
        "rv": "'of'"
      }
    ]
  }
]